test/cli_test.dart

```diff
@@ -15,7 +15,7 @@ void main() {
         initialBuffer.writeln(line);
       });
       initialContents = initialBuffer.toString();
-      output = command.start().lines;
```

Comment:
😅 shit happens when you work so late in the night

Reply:
Been there before! :smile: